services/explorer/src/pages/Home/Home.tsx

```diff
@@ -17,7 +17,7 @@ const HomePage = () => {
         title={currentNetwork}
         subtitle='Search for addresses, block hashes, transactions...'
       />
-      <ChartContainer />
+      <ChartContainer pbftBlocks={pbftBlocks} dagBlocks={dagBlocks} />
```

Comment:
Are we sure that for the charts we need the same pbft and dag blocks as we do for the cards? For the cards I am only using the last 10 blocks. But for charts, I don't know if we need just the last 10 or more. Can you please double-check this before merging?

Reply:
Yo guys I'm back. On the charts each vertical bar should represent a block, therefore you can visually count how much we display or even make it dynamic so it displays as much as it's given.

Reply:
@Malak67 @VargaElod23 I'm just passing pbftBlocks and dagBlocks entirely to the ChartContainer. We probably could be okay just passing them 6 elements of each array since we need the previous block for each block we are iterating to calculate block time.